ivy/ivy.xml

```diff
@@ -66,6 +66,13 @@
 		<!-- End of Hadoop Dependencies -->
 
 		<dependency org="org.apache.tika" name="tika-core" rev="1.17" />
+		<dependency org="net.sourceforge.owlapi" name="owlapi-api" rev="5.1.0" />
```

Comment:
The dependencies need to go into the plugin ivy.xml. They do not belong here.

Reply:
f02fb23